avo-hq/avo
# fix: attachment destroy button broken

# Description


Fixes #1337

Fixes an issue where the `confirm` and `method` attributes are wrongly attributed and the attachment destroy button is broken.

# Checklist:




## Manual review steps


1. Go to a project with a file attached
1. click on delete attachment button
2. click on cancel
3. observe how you are not redirected and the file is still there
4. cliek on delete attachment button again
5. click on ok
6. observe the attachment is destroyed

Manual reviewer: please leave a comment with output from the test if that's the case.
